pkg/aws/ec2/api/wrapper.go

```diff
@@ -306,6 +310,54 @@ var (
 		},
 	)
 
+	ec2DisassociateTrunkInterfaceCallCnt = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "ec2_disassociate_trunk_interface_api_req_count",
+			Help: "The number of calls made to EC2 to remove association between a branch and trunk network interface",
+		},
+	)
+
+	ec2DisassociateTrunkInterfaceErrCnt = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "ec2_disassociate_trunk_interface_api_err_count",
+			Help: "The number of errors encountered while removing association between a branch and trunk network interface",
+		},
+	)
+
+	VpcCniAvailableClusterENICnt = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "vpc_cni_created_available_eni_count",
+			Help: "The number of available ENIs created by VPC-CNI that will tried to be deleted by the controller",
```

Comment:
can we rename this to vpc_cni_created_pending_deletion_eni_count?. here it looks like just the number of eni created by vpc_cni.

Reply:
`available` here indicates the state of the network interface created by the corresponding component. "pending delete" is not accurate as the ENI may not be necessarily deleted if it is re-attached to an instance.